test/utils/wasm3_engine.cpp

```diff
@@ -64,9 +64,29 @@ bool Wasm3Engine::parse(bytes_view input)
     return true;
```

Comment:
I believe we don't need `LoadModule` and hence the runtime here, just `ParseModule`?

Reply:
This is done later.